src/dhtproto/node/neo/request/Mirror.d

```diff
@@ -383,6 +393,10 @@ public abstract scope class MirrorProtocol_v0
         this.value_buffer = this.resources.getVoidBuffer();
         this.update_queue.initialise(*this.resources.getVoidBuffer());
         this.refresh_queue.initialise(*this.resources.getVoidBuffer());
+        const max_batch_size = 64 * 1024; // TODO: read from config
```

Comment:
Double checking: this TODO should be merged at this stage, right?

Reply:
Yeah, this is something that'll be addressed in the future. (#75)